frontend/pages/SoftwarePage/SoftwareOS/SoftwareOSTable/SoftwareOSTable.tsx

```diff
@@ -40,8 +43,27 @@ interface ISoftwareOSTableProps {
   teamId?: number;
   isLoading: boolean;
   resetPageIndex: boolean;
+  platform?: SelectedPlatform;
 }
 
+const PLATFORM_FILTER_OPTIONS = [
+  {
+    disabled: false,
+    label: "All platforms",
+    value: "all",
+  },
+  {
+    disabled: false,
+    label: "macOS",
+    value: "darwin",
+  },
+  {
+    disabled: false,
+    label: "Windows",
+    value: "windows",
+  },
```

Comment:
Ahh, I see the issue with linux
It looks like the API (`https://dogfood.fleetdm.com/api/latest/fleet/os_versions?order_key=hosts_count&order_direction=desc&page=0&per_page=20&platform=chrome)` can return `platform: "chrome"`, `platform: "ipados"`, and `platform: "ios"` though so we should add that?

Reply:
To summarize, linux platforms are separated where the `platform` key is more specific to the distribution (example `platform: "rhel"` in screenshot) so we can't quite add "Linux" as a dropdown option. 

<img width="672" alt="Screenshot 2024-09-04 at 10 11 33 AM" src="https://github.com/user-attachments/assets/68396ab8-030e-4122-be51-e069fbc1f7c5">

Question for product: Do you want us to ignore linux platforms in this dropdown?

Reply:
cc: @rachaelshaw

Reply:
I thought about that, but those OS versions are unsupported for OS version based vuln scanning (just like Linux) so there's a bit less value there, and it would make Linux a more glaring omission, vs. currently "you can filter by platforms that we can check version vulns for".

We could have a follow-on for this that both adds Linux support and adds the other platforms to the selector though?

Reply:
@iansltx good point from a vuln scanning perspective

Let's confirm with product/design? cc: @randy-fleet